llvm/unittests/MI/LiveIntervalTest.cpp

```diff
@@ -156,6 +156,7 @@ static void testHandleMoveIntoNewBundle(MachineFunction &MF, LiveIntervals &LIS,
 
   // Build bundle
   finalizeBundle(MBB, I, std::next(ToInstr.getIterator()));
+  MF.getProperties().reset(MachineFunctionProperties::Property::IsSSA);
```

Comment:
The test probably shouldn't have had IsSSA in the first place

Reply:
IsSSA is computed by the MIRParser

Reply:
By buggy I mean assume SSA by default runs into these sorts of issues. Tests have a definite SSAness which it gets wrong. You could just as well clear this before finalizeBundle

Reply:
I guess @yashssh ran into a similar situation once and was trying to fix the default isSSA property assumed by MIRParser. Wasn't that ever landed?